Contributor documentation - setup, adding components, releasing (#946)
primer/view_components: contributor-docs/README.md

# Contributing

Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.

> If you have any substantial changes that you would like to make, please [open an issue](issues/new) first to discuss them with us.

Reviewer: ```suggestion
If you have any substantial changes that you would like to make, please [open an issue](issues/new) to start a discussion.
```

Thought not including "us" at the end makes it sound more community friendly.
